substrate/frame/election-provider-multi-block/src/verifier/tests.rs

```diff
@@ -444,19 +444,41 @@ mod async_verification {
 
 			roll_next();
 
-			// we instantly stop.
-			assert_eq!(verifier_events(), vec![Event::<Runtime>::VerificationDataUnavailable]);
+			// After first roll, only page 2 is processed, status is still Ongoing(1).
+			assert_eq!(verifier_events(), vec![Event::<Runtime>::Verified(2, 0)]);
+			assert_eq!(VerifierPallet::status(), Status::Ongoing(1));
+
+			// Process the next page (page 1).
+			roll_next();
+			assert_eq!(
+				verifier_events(),
+				vec![Event::<Runtime>::Verified(2, 0), Event::<Runtime>::Verified(1, 0)]
+			);
+			assert_eq!(VerifierPallet::status(), Status::Ongoing(0));
+
+			// Process the final page (page 0).
+			roll_next();
+			// Check that the verification has failed.
+			assert_eq!(
+				verifier_events(),
+				vec![
+					Event::<Runtime>::Verified(2, 0),
+					Event::<Runtime>::Verified(1, 0),
+					Event::<Runtime>::Verified(0, 0),
+					Event::<Runtime>::VerificationFailed(0, FeasibilityError::InvalidScore),
```

Comment:
do you think it is safe to also treat the lack of score as okay, make it `.unwrap_or_default()`, or immediately return `Rejected`? 

Since we have a notion of `minimumScore`, it should be fine,as long as we remember to set it on Kusama :))